Woof/Woof/View/OwnerCardView/OwnerCardView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+/// A view displaying information about the pet owner.
+struct OwnerCardView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: OwnerCardViewModel
```

Comment:
Do we really need the viewModel for this view?
It's just a simple view to group some elements.

Reply:
I thought, that there is a rule: one view = one view model.

Reply:
The viewModel are deleted.